guide_software_engineer.md

```diff
@@ -45,6 +46,13 @@ Hacker Rank の[Problem Solving(Basic)](https://www.hackerrank.com/skills-verifi
 
 主に、募集要項の必須条件を満たしているかを確認します。Coding Test の結果とあわせて総合的に判断を行います。
 
+## ML 選考課題
+
+ソフトウェアエンジニア（ML）へ応募した方のみを対象とし、ML モデル作成に関する課題に取り組んでいただきます。所要時間として1~2日程度かかることを想定しています。<br>
```

Comment:
I want to know your opinion.
Do you think the candidate should take a few days to the task?
I think a few days is too heavy and 4, or 5 hours is enough for it.